pkg/e2e/compose_test.go

```diff
@@ -125,8 +125,28 @@ func TestComposePull(t *testing.T) {
 	res := c.RunDockerComposeCmd(t, "--project-directory", "fixtures/simple-composefile", "pull")
 	output := res.Combined()
 
-	assert.Assert(t, strings.Contains(output, "simple Pulled"))
-	assert.Assert(t, strings.Contains(output, "another Pulled"))
+	expected := []string{
```

Comment:
nit: could we make this a bit more deterministic? Playing around with it locally, seems like `Skipped - Image is already being pulled by` is the only `Skipped - xxx` message that needs checking, and then maybe `Pulled`. Maybe we could do away with `contains()` entirely and just check the output as a whole contains those key phrases

Reply:
I believe I have an idea how to make this more explicit

Reply:
done